src/lib/storage/column_iterables.hpp

```diff
@@ -83,9 +85,9 @@ class Iterable {
 };
 
 /**
- * @brief base class of all indexable iterables
+ * @brief base class of all point-accessible column iterables
```

Comment:
Can we keep it “point access”, please? There is also an iterator which is called `PointAccessIterator` and it’s definitely not a `RandomAccessIterator`. This has a different meaning.

Reply:
Could you explain the difference between the two concepts? I am going by the STL definition:

"A RandomAccessIterator is a BidirectionalIterator that can be moved to point to any element in constant time. "

http://en.cppreference.com/w/cpp/concept/RandomAccessIterator

Reply:
Exactly, the random access iterator implements a certain interface, namely things like `it += 5`, which the point-access iterator does not. I was thinking about refactoring the point-access iterators at a later point, changing them into random access iterators, because actually that’s what they are, a random access iterator + a position list. Right now, calling them random-access iterator would be confusing.

Reply:
got it